### PR TITLE
CORE-215/CORE-475 feature: trigger a customized notification for succ…

### PR DESCRIPTION
**Pull request by lakhbawa**
…essful flow container import

### CODE REVIEW

File: src/views/ImportFlow.vue [hunk 1]
```diff
@@ -218,9 +218,23 @@ class ImportFlow extends mixins(Lang, Routes) {
       persistRoute: this.route('flows.persistFlowImport', {}),
       flowContainer: this.flowContainer,
     })
+    const numberOfFlows = get(flowContainer, 'flows.length', 0);
 
     if (flowContainer != null) {
       this.reset()
+      if (numberOfFlows > 1) {
+          this.$toast.success(`${Lang.trans('flow-builder.multiple-flows-saved', {
+          number: numberOfFlows,
```
Comment (safydy-r) [inline src/views/ImportFlow.vue:227]: missing indentation

File: src/views/ImportFlow.vue [hunk 2]
```diff
@@ -218,9 +218,23 @@ class ImportFlow extends mixins(Lang, Routes) {
...
+          })}`, {
+            timeout: 3000,
+            hideProgressBar: true,
+          })
+      } else {
+        this.$toast.success(`${Lang.trans('flow-builder.flow-saved')}`, {
+            timeout: 3000,
+            hideProgressBar: true,
+          })
+      }
```
Comment (safydy-r) [inline src/views/ImportFlow.vue:237]: I'd suggest we format the message in the condition, and fire the toast after the condition
```
if () {
   message = trans('myMessage1')
} else {
   message = trans('myMessage2')
}

this.$toast.success(message, ...)
```

- [ ] refactor to call the toast once

File: src/views/ImportFlow.vue [hunk 3]
```diff
@@ -218,9 +218,23 @@ class ImportFlow extends mixins(Lang, Routes) {
       persistRoute: this.route('flows.persistFlowImport', {}),
       flowContainer: this.flowContainer,
     })
+    const numberOfFlows = get(flowContainer, 'flows.length', 0);
```
Comment (safydy-r) [inline src/views/ImportFlow.vue:221]: We don't put `;` for JS codes

- [ ] make the end of line uniform, remove `;`

Comment (bulatgab) [inline src/views/ImportFlow.vue:221]: Maybe let's avoid `get` and use a dot chaining instead, followed by `?? defaultValue` if needed.

**Review by bulatgab — APPROVED** [review]
LGTM 👍 
with minor suggestions

File: src/views/ImportFlow.vue [hunk 4]
```diff
@@ -214,16 +214,29 @@ class ImportFlow extends mixins(Lang, Routes) {
...
   async handleImportFlow(route: string) {
     const flowContainer = await this.flow_persistImport({
-      // @ts-ignore - Would need to switch mixins to class components to fix this - https://class-component.vuejs.org/guide/extend-and-mixins.html#mixins
+      // Would need to switch mixins to class components to fix this - https://class-component.vuejs.org/guide/extend-and-mixins.html#mixins
       persistRoute: this.route('flows.persistFlowImport', {}),
       flowContainer: this.flowContainer,
     })
+    const numberOfFlows = flowContainer.flows.length ?? 0
 
-    if (flowContainer != null) {
```
Comment (lakhbawa) [inline src/views/ImportFlow.vue:222]: flow container is never null so this condition would always be true

**Comment by lakhbawa** [comment]
ready for second review @safydy-r @bulatgab

**Review by safydy-r — APPROVED** [review]
lgtm, thanks

**Review by bulatgab — APPROVED** [review]
Approving. Thanks!

**Pull request merged**